ui/src/plugins/Dialog.json

```
@@ -307,7 +307,7 @@
             },
 
             "component": {
-              "type": "Component",
+              "type": ["Component","String"],
```

To match with the existing code style.
```suggestion
              "type": [ "Component", "String" ],
```